lib/services/page-service.ts

```diff
@@ -38,47 +42,24 @@ export class PageService implements IPageService {
         });
     }
 
-    private ensurePageTemplate(pageName: string, flavor: string): Promise<string> {
-        const templatesDir = util.path.join(__dirname, "../templates");
-        const pagePath = util.path.join(templatesDir, pageName);
-
-        return new Promise((resolve, reject) => {
-            util.fs.ensureDir(templatesDir)
-                .then(() => {
-                    return util.pageExists(templatesDir, pageName);
-                })
-                .then((pageExists: boolean) => {
-                    if (!pageExists) {
-                        return this.clonePageTemplate(pageName, flavor, templatesDir);
-                    }
-                })
-                .then(() => {
-                    resolve(pagePath);
-                })
-                .catch((ensurePageError: any) => {
-                    reject(ensurePageError);
-                });
-        });
-    }
-
-    private clonePageTemplate(pageName: string, flavor: string, templatesDir: string): Promise<string> {
+    private clonePageTemplate(pageName: string, flavor: string, templatesDirectory: string): Promise<string> {
         const command = "git";
-        const commandArgs: Array<any> = ["clone"];
+        const commandArguments: Array<any> = ["clone"];
 
         return new Promise((resolve, reject) => {
             util.getPageTemplatesBaseUrl(flavor)
                 .then((baseUrl: string) => {
                     baseUrl = baseUrl + ".git";
```

Comment:
Not sure we need to reassign the `baseUrl` param here, instead:

```
commandArguments.push(`${baseUrl}.git`);
```

Reply:
Different PR?